docs/spl-docs/evaluation-functions/multivalue-functions.md

```diff
@@ -0,0 +1,19 @@
+# Multivalue functions
+
+_This list includes functions that can be used to return multivalue fields or can operate on multivalue fields._
+
+
+
+## split(\<str\>, \<delim\>)
+This function splits `<str>` with the delimiter `<delim>` and return these parts as a multivalue field.
+
+#### Usage
+- `<str>` argument can be a string literal or a field name. 
+- `<delim>` is a string literal specifying the delimiter. 
+- One can use this function with an eval command.
```

Comment:
> `<str>` argument can be

nit: `<str> argument can be ...` doesn't quite make sense. Either `The <str> argument can be ...` or `<str> can be ...` would be better. But since "the argument" doesn't add information, let's choose the more concise version:
```
`<str>` can be ...
```

There's several other occurrences of this in several of your files that you should change similarly